src/main/java/net/openhft/chronicle/threads/NotifyDiskLowLogWarn.java

```diff
@@ -0,0 +1,20 @@
+package net.openhft.chronicle.threads;
+
+import net.openhft.chronicle.core.Jvm;
+
+import java.nio.file.FileStore;
+
+public class NotifyDiskLowLogWarn implements NotifyDiskLow {
+    @Override
+    public void panic(FileStore fileStore) {
+        Jvm.error().on(DiskSpaceMonitor.class, "your disk " + fileStore + " is almost full, " +
+                "warning: the JVM may crash if it undertakes an operation with a memory-mapped file.");
+    }
+
+    @Override
+    public void warning(double diskSpaceFullPercent, FileStore fileStore) {
+        Jvm.error().on(DiskSpaceMonitor.class, "your disk " + fileStore
```

Comment:
I meant Jvm.error() for the `panic()` only. This one should stay `warn()`.

Reply:
Yes - that sounds good. Done1